internal/util/http.go

```diff
@@ -25,6 +25,17 @@ import (
 	"github.com/coreos/ignition/internal/version"
 )
 
+// A timeout of 0 indicates no timeout
+var (
+	httpResponseHeaderTimeout int
```

Comment:
What other options are there in this case? The timeouts are applied globally. The only other alternative I see is passing either them (or the Timeouts struct) to every function that might use make http requests, which also doesn't seem very clean